packages/core/src/blitz-script.tsx

```diff
@@ -2,11 +2,18 @@ import {NextScript} from "next/document"
 import React from "react"
 import {BlitzData} from "./blitz-data"
 
-export function BlitzScript() {
+export type BlitzScriptProps = {
+  /** nonce property to pass to the <script> element generated by Blitz. */
+  nonce?: string
+  /** crossOrigin property to pass to the <script> element generated by Blitz. */
+  crossOrigin?: string
+}
+
+export function BlitzScript({nonce, crossOrigin}: BlitzScriptProps) {
   return (
     <>
       <BlitzData />
-      <NextScript />
+      <NextScript nonce={nonce} crossOrigin={crossOrigin} />
     </>
   )
 }
```

Comment:
I think we should just pass all props to NextScript, what do you think?

And BlitzScriptProps can inherit from `ComponentPropsWithoutRef<typeof NextScript>`